ENGAGEHF/Questionnaire/QuestionnaireSheetView.swift

```diff
@@ -40,37 +43,43 @@ struct QuestionnaireSheetView: View {
                     }
 #endif
                     
-                    do {
-                        try await standard.add(response: questionnaireResponse)
-                        dismiss()
-                    } catch {
-                        viewState = .error(AnyLocalizedError(error: error))
+                    Task {
+                        do {
+                            try await standard.add(response: questionnaireResponse)
+                            try? await Task.sleep(nanoseconds: 1_000_000_000)
+                            dismiss()
+                        } catch {
+                            viewState = .error(AnyLocalizedError(error: error))
+                        }
                     }
                 }
             } else {
                 ProgressView("Questionnaire Loading")
                     .background(Color(.systemGroupedBackground))
             }
         }
-            .task {
-                do {
+        .task {
```

Comment:
It would be amazing to try to keep the diff as small as possible to make it easier to review and therefore avoid indentation diffs.